### DISCUSSION TITLE
Tried import - date fields failing

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#16651] [How to]
Hi, I am importing some invoices and it happen that dates are not imported. User (admin) date format is dd-mm-yyyy and source csv date fields are also dd-mm-yyyy. The imported records has blank date fields. What am I missing?

**Comment by mariuszkrzaczkowski** [comment]
Have you checked the logs? 
maybe you have some whitespace and validation doesn't pass

**Reply by garryatbrighton** [reply]
Got it. For import (at least here) date source format should be yyyy-mm-dd and than all gets set OK.
About to finish migration of 9yo vtiger. Took me 2 weeks (lightly) and teach a lot about tricks...
